src/components/LinkManager/LinkList/LinkList.tsx

```diff
@@ -0,0 +1,17 @@
+import { Link } from "@/src/constants/types";
+import LinkOverview from "./LinkOverview/LinkOverView";
+import styles from "./LinkList.module.scss";
+
+interface LinkList {
+  links: Link[];
+}
+
+export default function LinkList({ links }: LinkList) {
+  return (
+    <section className={styles.linkList}>
+      {links?.map((link) => (
```

Comment:
links값이 무조건 있다고 타입이 정의되어 있기 때문에 옵셔널 체이닝은 필요없을 것 같습니다~

```suggestion
      {links.map((link) => (
```